charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -84,13 +61,13 @@ controller:
   nodeSelector: {}
   podAnnotations: {}
   podLabels: {}
-  priorityClassName:
+  priorityClassName: system-cluster-critical
   # AWS region to use. If not specified then the region will be looked up via the AWS EC2 metadata
   # service.
   # ---
   # region: us-east-1
   region:
-  replicaCount:
+  replicaCount: 2
   resources: {}
   containerResources:
```

Comment:
WDYT about renaming the sidecars.*image keys to be sidecar.*.image, and then adding sidecar.*.resources, to be consistent with EFS? https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/charts/aws-efs-csi-driver/values.yaml#L21